test/coding/test_commandline_code_executor.py

```diff
@@ -218,3 +218,24 @@ def test_valid_relative_path(cls) -> None:
         assert "test.py" in result.code_file
         assert (temp_dir / "test.py").resolve() == Path(result.code_file).resolve()
         assert (temp_dir / "test.py").exists()
+
+
+@pytest.mark.parametrize("cls", classes_to_test)
+@pytest.mark.parametrize("lang", ["bash", "shell", "sh"])
+def test_silent_pip_install(cls, lang: str) -> None:
+    if sys.platform in ["win32"]:
```

Comment:
For skipping windows test we can use @pytest.skipif(... ). Why do we want to skip on windows?

Reply:
we can add powershell language right?

Reply:
Yeah, I didn't have a way to test it locally. I added a check if windows, skip unix shells, and vice versa.